configure.ac

```diff
@@ -370,6 +370,7 @@ AC_CHECK_HEADERS([ \
 	stdint.h \
 	stdio_ext.h \
 	stdlib.h \
+	sys/auxv.h \
```

Comment:
Are there systems where this header is missing?

Reply:
I'm unsure; it's GNU stuff, not any official standard. Better safe than sorry ;-)